docs/catalog-information/query-examples.md

```diff
@@ -63,6 +63,46 @@ above example declare a query for `mc:productId` field in catalog and response w
 `startPosition`  will display records from all result start from the first one to the `maxRecords` value
 ***(1 to 1 in example)***.
 
+## Query by ***IsEqualTo*** value
+How to query the catalog by **specific property** and his **value**
+```xml
+POST request
+
+Url:
+http://localhost:8080/csw
+
+Content-type:
```

Comment:
Correct header typing is `Content-Type`, not `Content-type`
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

Reply:
fixed